src/cascadia/TerminalSettingsModel/AIConfig.idl

```diff
@@ -21,8 +22,10 @@ namespace Microsoft.Terminal.Settings.Model
         String AzureOpenAIKey;
         static event AzureOpenAISettingChangedHandler AzureOpenAISettingChanged;
 
-
         String OpenAIKey;
-        static event OpenAISettingChangedHandler OpenAISettingChanged;
+        static event OpenAISettingChangedHandler OpenAISettingChanged;
+
+        String GithubCopilotAuthToken;

```

Comment:
can we just treat the ValueSet as fully opaque and store the whole thing in the credential store?

rather than cracking it and having Terminal read/write it as individual keys